src/metadata.rs

```diff
@@ -378,7 +379,7 @@ impl ComponentMetadata {
         }
 
         Ok(Self {
-            name: package.name.clone(),
+            name: package.name.clone().to_kebab_case(),
```

Comment:
The package name in the metadata should match the name from the cargo metadata, ideally.

Instead, I think this fix needs to be applied to https://github.com/bytecodealliance/cargo-component/blob/main/src/bindings.rs#L434 (replacing the `clone` call).

Reply:
Thanks! Updated and added a build test.